{"name": "测试数据：【 测试test，True，0，删除用户信息成功 】", "status": "broken", "statusDetails": {"message": "AttributeError: 'NoneType' object has no attribute 'get'", "trace": "self = <testcases.api_test.test_05_delete_user.TestUserDelete object at 0x10412a9a0>\nlogin_fixture = {'code': 1003, 'msg': '用户名不存在！！！'}, username = '测试test'\nexcept_result = True, except_code = 0, except_msg = '删除用户信息成功'\n\n    @allure.story(\"用例--删除用户信息\")\n    @allure.description(\"该用例是针对获取用户删除接口的测试\")\n    @allure.issue(\"https://blog.csdn.net/\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://blog.csdn.net/\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"测试数据：【 {username}，{except_result}，{except_code}，{except_msg} 】\")\n    @pytest.mark.single\n    @pytest.mark.parametrize(\"username, except_result, except_code, except_msg\",\n                             api_data[\"test_delete_user\"])\n    @pytest.mark.usefixtures(\"insert_delete_user\")\n    def test_delete_user(self, login_fixture, username, except_result, except_code, except_msg):\n        logger.info(\"*************** 开始执行用例 ***************\")\n        user_info = login_fixture\n>       admin_user = user_info.get(\"login_info\").get(\"username\")\nE       AttributeError: 'NoneType' object has no attribute 'get'\n\ntestcases/api_test/test_05_delete_user.py:36: AttributeError"}, "description": "该用例是针对获取用户删除接口的测试", "attachments": [{"name": "log", "source": "67279957-158e-4fd9-9e9f-3c6fdcb7b829-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "46435a64-fe0c-4b0a-bd1b-4bd8559266ed-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'测试test'"}, {"name": "except_result", "value": "True"}, {"name": "except_code", "value": "0"}, {"name": "except_msg", "value": "'删除用户信息成功'"}], "start": 1744708530331, "stop": 1744708530333, "uuid": "17ba277c-1380-4313-91be-622312962fcf", "historyId": "da319e4398ddda3c7eb597b8bd4ec2fc", "testCaseId": "f140003f8969cec64326a77ad093297e", "fullName": "testcases.api_test.test_05_delete_user.TestUserDelete#test_delete_user", "labels": [{"name": "epic", "value": "针对单个接口的测试"}, {"name": "severity", "value": "normal"}, {"name": "story", "value": "用例--删除用户信息"}, {"name": "feature", "value": "用户删除模块"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "testcases.api_test"}, {"name": "suite", "value": "test_05_delete_user"}, {"name": "subSuite", "value": "TestUserDelete"}, {"name": "host", "value": "192.168.10.5"}, {"name": "thread", "value": "24590-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api_test.test_05_delete_user"}], "links": [{"type": "test_case", "url": "https://blog.csdn.net/", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://blog.csdn.net/", "name": "点击，跳转到对应BUG的链接地址"}]}